### DISCUSSION TITLE
看一看我国自杀率变化趋势 | /project/population3/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#616] [Announcements]
https://yuanfan.rbind.io/project/population3/

**Comment by yanyuteng** [comment]
谢谢博主，这篇文章很有意思，做了我一直想看的分年龄段趋势

**Reply by earfanfan** [reply]
这位小伙伴，你好，你还有其他想看的内容，或者其他想看的主题吗？你愿意的话，可以写下来给我参考参考。

**Reply by yanyuteng** [reply]
蹲一下您之前写的定西和scihub类型的文章，持续关注嘿嘿

**Comment by Z-NJ** [comment]
宝藏博主！

**Reply by earfanfan** [reply]
咦，天上掉下来一个荣誉勋章，我捡到啦，谢谢您。

**Comment by L1Xu4n** [comment]
你好，可以统计一下最近全国各省份自杀率吗？

**Reply by earfanfan** [reply]
这位小伙伴，你好，这个问题我有考虑过，不过我在网上没有找到合适的数据源，要是将来能找到的话，我会收集这部分数据的。

**Reply by yanyuteng** [reply]
目前应该只有分东中西及城乡的分死因数据，可以参考疾控中心出版的[《中国死因监测数据集》](https://ncncd.chinacdc.cn/jcysj/siyinjcx/syfxbg/)。更细的数据要去找分省市疾控中心或流行病学杂志上的相关论文，这个整理的工作量太大了。

**Comment by Jinchhu** [comment]
老师您好，我可以在我的毕业论文中使用您的图和代码吗？

**Reply by earfanfan** [reply]
您好，可以呀。

**Reply by Jinchhu** [reply]
感谢老师，这些图做的太好啦~！！

**Comment by xszong** [comment]
单从数据看。国内老年人自杀的数据比其他美日高很多。

**Reply by earfanfan** [reply]
诶，是凝光啊。其实就算不比较国家之间，仅仅看文中第二张图也可以看出来，农村老年人自杀率比城市高很多。

**Comment by suaddy698-lang** [comment]
老师好，请问我可以将您的图和代码翻译后用于课程论文吗？非常感谢！

**Reply by earfanfan** [reply]
可以的，你用吧。

**Reply by suaddy698-lang** [reply]
谢谢老师！！

**Comment by xszong** [comment]
我没什么意见





------------------ 原始邮件 ------------------
发件人: suaddy698-lang ***@***.***&gt;
发送时间: 2026年3月1日 19:27
收件人: earfanfan/yuanfan ***@***.***&gt;
抄送: xszong ***@***.***&gt;, Comment ***@***.***&gt;
主题: Re: [earfanfan/yuanfan] 看一看我国自杀率变化趋势 | /project/population3/ (Discussion #616)
